convert2rhel/unit_tests/actions/system_checks/package_updates_test.py

```diff
@@ -83,61 +83,20 @@ def test_check_package_updates_not_up_to_date(pretend_os, monkeypatch, package_u
     monkeypatch.setattr(package_updates, "get_total_packages_to_update", value=lambda: packages)
 
     package_updates_action.run()
-    unit_tests.assert_actions_result(
-        package_updates_action,
-        level="OVERRIDABLE",
-        id="OUT_OF_DATE_PACKAGES",
-        title="Outdated packages detected",
-        description="Please refer to the diagnosis for further information",
-        diagnosis=diagnosis,
-        remediations=(
-            "If you wish to ignore this message, set the environment variable "
-            "'CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP' to 1."
-        ),
-    )
-
-    assert diagnosis in caplog.records[-1].message
-
-
-@centos8
-def test_check_package_updates_not_up_to_date_skip(pretend_os, monkeypatch, package_updates_action):
-    packages = ["package-2", "package-1"]
-    diagnosis = (
-        "The system has 2 package(s) not updated based on repositories defined in the system repositories.\n"
-        "List of packages to update: package-1 package-2.\n\n"
-        "Not updating the packages may cause the conversion to fail.\n"
-        "Consider updating the packages before proceeding with the conversion."
-    )
-    monkeypatch.setattr(package_updates, "get_total_packages_to_update", value=lambda: packages)
-    monkeypatch.setattr(
-        os,
-        "environ",
-        {"CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP": 1},
-    )
-
     expected = set(
         (
             actions.ActionMessage(
                 level="WARNING",
-                id="SKIP_OUTDATED_PACKAGE_CHECK",
-                title="Skip package not up to date check",
-                description=(
-                    "Detected 'CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP' environment variable, we will skip "
-                    "the package up-to-date check.\n"
-                    "Beware, this could leave your system in a broken state."
-                ),
-            ),
-            actions.ActionMessage(
-                level="WARNING",
-                id="OUTDATED_PACKAGE_MESSAGE",
+                id="OUT_OF_DATE_PACKAGES",
                 title="Outdated packages detected",
                 description="Please refer to the diagnosis for further information",
                 diagnosis=diagnosis,
                 remediations="Run yum update to update all the packages on the system.",
+                variables={},
             ),
         )
     )
-    package_updates_action.run()
```

Comment:
I believe this was removed by mistake?

Reply:
The formatting just looks like that because I modified the test from being an assert action result to asserting an action message set. The run call is there present - https://github.com/oamg/convert2rhel/blob/b8d3c5734561bf0ea4333215d516db2034b4752c/convert2rhel/unit_tests/actions/system_checks/package_updates_test.py#L127